src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState, useMemo } from 'react';
+import debounce from 'lodash.debounce';
```

Comment:
if you added a new dependency in the `package.json` (lodash.debounce) you have to add `package.json` file to git and push it as well:
![image](https://github.com/mate-academy/react_autocomplete/assets/86923089/9a84dbb8-a22b-4a0a-9831-a46b0407582b)